### DISCUSSION TITLE
Scene Viewer: is transparency / transmission supported?

### DISCUSSION THREAD

**Discussion by freder** [google/model-viewer#3051] [Q&A]
hi,

where can I find more information on the support of transparency and transmission in `scene viewer`?

## my problem:
I have a `GLTF 2.0` model with a glass material that uses the `KHR_materials_transmission` extension. the material is one-sided and the alpha mode is `BLEND`.

it renders fine in any GLTF model viewer I've tried, but `scene viewer` itself seems to render the material opaque. – at least that's what I can tell from a handful of screenshots android users have provided.

the validation report on https://modelviewer.dev/editor/ seems to be ok.

- is transparency / transmission supported in `scene viewer`? 
- or does it potentially depend on the device, android version, or `ARCore` version?

thanks a lot in advance!

**Comment by elalish** [comment]
@AdrianAtGoogle @tpsiaki for scene viewer detail.
